lib/piwik/events.rb

```diff
@@ -12,4 +12,40 @@ class Events < ApiModule
       getCategoryFromNameId
     }
   end
-end
\ No newline at end of file
+
+  def category params = {}
```

Comment:
any reason why you skipped the parentheses here? are you trying to follow the style of the project that I cannot see in this PR? if not I'd say we stick to parens, which is what we use in the other ruby projects

Reply:
Just using same style as rest of the project. We can add to the entire project in a separate PR.